### PR TITLE
Replaces goto with go-to for consistency

### PR DESCRIPTION
**Pull request by bruno-oliveira**
Unifies commits

### CODE REVIEW

**Comment by bruno-oliveira** [comment]
@sbauer322 Is this what you intended? I hope I did it right!

**Comment by sbauer322** [comment]
Yes, @bruno-oliveira, that is about right - thank you! All the changes appear in this pull request, which is desired and much easier to review.

It looks like some extra files were added, namely the ones ending with a `~`. Could you remove those as they are unnecessary?

File: src/lt/objs/cli.cljs~ [hunk 1]
```diff
@@ -0,0 +1,75 @@
+(ns lt.objs.cli
```
Comment (sbauer322) [inline src/lt/objs/cli.cljs~]: This file can be removed.

File: src/lt/objs/find.cljs~ [hunk 2]
```diff
@@ -0,0 +1,229 @@
+(ns lt.objs.find
```
Comment (sbauer322) [inline src/lt/objs/find.cljs~]: This file can be removed.

File: src/lt/objs/search.cljs~ [hunk 3]
```diff
@@ -0,0 +1,277 @@
+(ns lt.objs.search
```
Comment (sbauer322) [inline src/lt/objs/search.cljs~]: This file can be removed.

File: src/lt/objs/find.cljs [hunk 4]
```diff
@@ -215,8 +215,8 @@
 
 (object/add-behavior! line-input ::exec-active!)
 
-(cmd/command {:command :goto-line
-              :desc "Editor: Goto line"
+(cmd/command {:command :go-to-line
+              :desc "Editor: go to line"
```
Comment (sbauer322) [inline src/lt/objs/find.cljs]: This line should have a capital 'G' for consistency... so `:desc "Editor: Go to line"`

**Comment by bruno-oliveira** [comment]
Hello, 
I have cleaned up the repository and also changed the "go to line" to "Go to line" for consistency, as requested.

I hope everything is well now, and if this is merged, I'd love to start helping with other issues suitable for beginners!

**Comment by sbauer322** [comment]
Tested it out and there is one more necessary minor change.

As the command `:goto-line` was renamed, it will be necessary to update the corresponding default keymap entry to use `:go-to-line` instead. This is located at `deploy/settings/default/default.keymap` on line 26.

Otherwise, everything seems in order.

**Comment by bruno-oliveira** [comment]
Hello,
I'll update the keymap as soon as possible! Definitely today still!

Em 31/12/2016 03:18, "Scott Bauer" <notifications@github.com> escreveu:

**Comment by bruno-oliveira** [comment]
Hello @sbauer322 ,
I have just updated the keymap as requested and also deleted an additional file that was generated ending in ~. I hope everything is finally in order! I'll be happy to fix whatever needs fixing still!

**Comment by bruno-oliveira** [comment]
Any good news? :)

**Comment by sbauer322** [comment]
This looks good to me - `Ctrl-L` is working again! Thank you for your efforts!

The final thing you will need to do is squash your commits into a single commit.

I am going to leave this open until January 4th in order to give other @LightTable/maintainers a chance to look it over, should they decide to.

**Comment by bruno-oliveira** [comment]
Hello,
I think I have squashed all previous commits into one, as requested... Is it all correct?

**Comment by sbauer322** [comment]
Yes, that seems correct. I hate to keep picking, but could you update the commit message to some more meaningful? When it gets merged to master, the current commit message won't be very helpful down the line.

**Comment by sbauer322** [comment]
Looks like it is in great shape!

**Comment by bruno-oliveira** [comment]
That's good to know! :D Thanks for all the feedback and help during the process

**Comment by bruno-oliveira** [comment]
Any updates on the state of merging? :)
I hope all is good, but, if I still need to do something, I will gladly do it!

**Comment by sbauer322** [comment]
Simply was giving anyone an opportunity to provide feedback... it is never a bad idea to have additional eyes on code!

Since no one has provided any additional feedback I will be merging this tomorrow, January 5th.

**Pull request merged**